tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -13,8 +13,10 @@ export const clickResource = async ({
 }): Promise<void> => {
   const paths = path.split('/')
   for (const name of paths) {
+    const resourceSelector = `[data-test-resource-name="${name}"]`
+    await page.waitForSelector(resourceSelector)
```

Comment:
Some time ago this step failed most of the time in CI and a few times locally because it won't get `textfile.txt` that was drop uploaded. Locally upon reloading the page the file would appear, couldn't reproduce it manually so I thought it was test timing problem so I added that line to be sure that the selector is preset before we do the click action.  
https://github.com/owncloud/web/blob/b4e626f293826099c284d6612716488a9cf6fe40/tests/e2e/cucumber/features/integrations/link.ocis.feature#L34-L37 

I haven't seen that step fail since then I'll remove it and see if it has the same behavior.